src/backend/api/controllers/reviewControllers.js

```diff
@@ -0,0 +1,147 @@
+const knex = require("../../database");
+
+const getAllReviews = async (req, res) => {
+    try {
+        const reviews = await knex
+        .from("review")
+        .select(
+            "id", 
+            "title",
+            "description",
+            "meal_id",
+            "stars",
+            "created_date"
+        );
+
+        reviews.length === 0
+        ? res.status(404).json({"error": "Reviews not found"})
+        : res.status(400).json(reviews);
```

Comment:
the status codes need to be updated. And I would suggest to just return the reviews object whether it contains the reviews or not.